mitogen/__init__.py

```diff
@@ -35,7 +35,7 @@
 
 
 #: Library version as a tuple.
-__version__ = (0, 2, 9)
+__version__ = (0 , 3, 0, 'rc', 0)
```

Comment:
This results in the version string "0.3.0.rc.0", which setuptools/pip normalises to "0.3.0rc0", to meet PEP440. For #801 I tweaked setup.py as well https://github.com/mitogen-hq/mitogen/pull/801/commits/9b1617f26f03d6b90065bbdb71412c216f3c2e1a

As far as the outside world is concerned both are good, so LGTM.